apps/api/src/app/events/usecases/send-message/send-message-type.usecase.ts

```diff
@@ -36,3 +47,17 @@ export abstract class SendMessageType {
     );
   }
 }
+
+function toStringify(error: any): string | boolean {
+  if (!error) return false;
+
+  if (typeof error === 'string' || error instanceof String) {
+    return error.toString();
+  }
+
+  if (Object.keys(error)?.length > 0) {
+    return JSON.stringify(error);
+  }
+
+  return false;
+}
```

Comment:
What do you think of returning undefined instead of a boolean for the cases there is not possible to stringify the error? It will engage nicely with the fallback checks done in `sendErrorStatus` and simplifies the return type of the function.

Reply:
@p-fernandez Tend to agree now that I looking at it it looks more complicated than it should.